packages/react-dom/src/__tests__/ReactDOMFizzStaticBrowser-test.js

```diff
@@ -461,7 +461,7 @@ describe('ReactDOMFizzStaticBrowser', () => {
       if (prerendering) {
         React.unstable_postpone();
       }
-      return 'Hello';
+      return ['Hello', 'World'];
```

Comment:
We have another test that tests a single return value. This test ensures we revisit both paths and not just "slot 0".